framework/include/utils/RankTwoTensor.h

```diff
@@ -170,6 +170,9 @@ class RankTwoTensor
   RankTwoTensor operator - () const;
 
   /// performs _vals *= a
```

Comment:
Actually I'm pretty convinced that we should _not_ add this operator overload @tonkmr